src/Symfony/Component/Translation/TranslatableMessage.php

```diff
@@ -28,6 +28,8 @@ public function __construct(
 
     public function __toString(): string
     {
+        trigger_deprecation('symfony/translation', '7.4', 'Calling "%s()" is deprecated.', __METHOD__);
```

Comment:
missing composer.json entry for symfony/deprecation-contracts also
```suggestion
        trigger_deprecation('symfony/contracts', '3.6', 'Method "%s()" is deprecated.', __METHOD__);
```

Reply:
Why ? We're inside symfony/component/translation

Reply:
And there is already the `"symfony/deprecation-contracts": "^2.5|^3"` entry